src/lib/components/workflow/workflow-json-navigator.svelte

```diff
@@ -96,7 +100,11 @@
   {:else}
     {#key index}
       <CodeBlock
```

Comment:
Looks like this is existing behavior, but when toggling `View decoded event history` off and then back on again it looks like there is a page "flash" and a full UI re-render. Might be something we can improve...

![Screenshot 2023-10-23 at 3 10 13 PM](https://github.com/temporalio/ui/assets/15069288/e305ebfd-6f00-4e22-aff0-8ad4392a6786)

Reply:
Good call. We need to rerender the CodeBlock on decode toggle, but I changed the initial decodeValue string from empty string to stringifyWithBigInt(value). So the initial value is the event un-decoded, and then if it decodes successfully, replaces it with the decoded value. This way you don't get that flash which is the change in height from empty string to the decoded value. Much better experience.

Reply:
I also changed the label to "Decode Event History" to make them consistent